localization/languages/pt-BR.json

```diff
@@ -31,89 +31,89 @@
         /* custom !bangs
         these are some of the items that show up when you press ! in the searchbar.
         Each one of these strings describes what the command will do when you run it. */
-        "showMoreBangs": null, //missing translation
+        "showMoreBangs": "Mostrar mais...",
         "viewSettings": "Ver configurações",
         "viewReadingList": "Ver listas de leitura",
         "takeScreenshot": "Capturar tela",
-        "clearHistory": "Limpar todo o histórico",
-        "clearHistoryConfirmation": null, //missing translation
+        "clearHistory": "Limpar histórico",
+        "clearHistoryConfirmation": "Limpar todo o histórico e dados de navegação?",
         "switchToTask": "Trocar para a tarefa",
-        "createTask": "Criar uma tarefa",
-        "closeTask": null, //missing translation
-        "moveToTask": "Mover esta aba para uma tarefa",
-        "nameTask": null, //missing translation
-        "addBookmark": null, //missing translation
+        "createTask": "Criar tarefa",
+        "closeTask": "Fechar tarefa",
+        "moveToTask": "Mover esta guia para uma tarefa",
+        "nameTask": " Nomear tarefa",
+        "addBookmark": "Adicionar favorito",
 		"searchBookmarks": "Buscar nos favoritos",
-        "bookmarksAddTag": null, //missing translation
-		"bookmarksSimilarItems": null, //missing translation
-		"searchHistory": null, //missing translation
-        "importBookmarks": null, //missing translation
-		"exportBookmarks": null, //missing translation
+        "bookmarksAddTag": "Adicionar tag",
+		"bookmarksSimilarItems": "Itens semelhantes",
+		"searchHistory": "Buscar no histórico",
+        "importBookmarks": "Importar favoritos",
```

Comment:
Could you translate the full original string, "Import bookmarks from HTML file", here?